setup/create-wp-project/src/basics/command-line.js

```diff
@@ -5,6 +5,27 @@ const { exec } = require('promisify-child-process');
 // RangeError [ERR_CHILD_PROCESS_STDIO_MAXBUFFER]: stderr maxBuffer length exceeded
 const cloneTimeout = 45000;
 
+const checkNodeVersion = async () => exec(`node -v`);
+const checkComposerVersion = async () => exec(`composer -v`);
+const checkWPCliVersion = async () => exec(`wp --info`);
+const checkGitVersion = async () => exec(`git --version`);
+
+const checkRequirements = async () => {
+  const nodeVersion = await checkNodeVersion();
+  if (nodeVersion.stderr !== '') process.exit(0);
```

Comment:
will the user get the somekind of msg if the node is not instaled?

Reply:
If you don't have node installed, the script can't start. if you trigger `npx create-wp-project` or if you try to manually trigger the script with for example `node  ~/infinum/eightshift-frontend-libs/setup/create-wp-project/create-wp-project.js` without node installed you will get a message saying `command not found: npx` or `command not found: node`

Reply:
is there a way to add some kind of user friendly msg other than `command not found: node`?

Reply:
Just for clarity: if you have `npx` installed, you'll have node installed as well. You also won't be able to run this script if you don't have node installed (not taking into account Node-compatible runtimes, which are rare, but exist). I expect this only to fail in cases where `node` isn't in a user's `$PATH`.

As discussed below:

```suggestion
  if (nodeVersion.stderr !== '') process.exit(1);
```